WooCommerce/Classes/ViewRelated/Dashboard/Analytics Hub/Time Range/AnalyticsHubDayRangeSelection.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+
+final class AnalyticsHubDayRangeSelection: AnalyticsHubTimeRangeSelectionDelegate {
+    var currentTimeRange: AnalyticsHubTimeRange?
+    var previousTimeRange: AnalyticsHubTimeRange?
+    var currentRangeDescription: String?
+    var previousRangeDescription: String?
```

Comment:
Can we replace all `var`s in all Selection objects with `let`s?

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/8267/commits/1077d07c3186a793daffece808c0132f1cc787cb